Vostok.Hosting/Components/Metrics/MetricsBuilder.cs

```diff
@@ -54,6 +57,12 @@ public IVostokMetricsBuilder CustomizeSettings(Action<MetricContextConfig> setti
             return this;
         }
 
+        public IVostokMetricsBuilder EnrichAnnotationTags(MetricTags tags)

```

Comment:
All others builders have CustomizeX (see CustomizeLog) method.

Better to have here just CustomizeAnnotationEventSender and this code may be an extension.